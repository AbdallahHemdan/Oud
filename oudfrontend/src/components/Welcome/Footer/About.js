import React, { Component } from 'react';
import Navbar from '../Navbar/Navbar';

import './../welcome.css';
class About extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div className="container aboutSection">
          <div className="Help_text">
            <h1 className="AboutHead">About US </h1>
            <p className="AboutPara">
              With OUD, it’s easy to find the right music or podcast for every
              moment – on your phone, your computer, your tablet and more. There
              are millions of tracks and episodes on OUD. So whether you’re
              behind the wheel, working out, partying or relaxing, the right
              music or podcast is always at your fingertips. Choose what you
              want to listen to, or let OUD surprise you. You can also browse
              through the collections of friends, artists, and celebrities, or
              create a radio station and just sit back. Soundtrack your life
              with OUD. Subscribe or listen for free.
            </p>
            <h1 className="AboutHead">Customer Service and Support</h1>
            <p>
              <ul>
                <li className="AboutPara">
                  <a href="/help">Help site.</a> Check out our help site for
                  answers to your questions and to learn how to get the most out
                  of OUD and your music.
                </li>
                <li className="AboutPara">
                  Community. Get fast support from expert OUD users. If there
                  isn't already an answer there to your question, post it and
                  someone will quickly answer. You can also suggest and vote on
                  new ideas for OUD or simply discuss music with other fans.
                </li>
                <li className="AboutPara">
                  <a href="/contactus">Contact us.</a> Contact our Customer
                  Support if you don't find a solution on our support site or
                  Community.
                </li>
              </ul>
            </p>
          </div>
        </div>
      </div>
    );
  }
}


export default About;