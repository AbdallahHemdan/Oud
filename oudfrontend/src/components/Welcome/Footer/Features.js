import React, { Component } from 'react';
import Navbar from '../Navbar/Navbar';
import './../welcome.css';
class Features extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div className="container HelpSection">
          <div className="Help_text">
            <h1>Features</h1>
            <div className="container">
              <div class="grid-container">
                <div class="grid-item">
                  <h2>Search</h2>
                  <h4>
                    Search a song, album, artist, label, genre, mood, activity,
                    or friend on Spotify.
                  </h4>
                </div>
                <div class="grid-item">
                  <h2>Playlists</h2>
                  <h4>
                    Playlists are like mixtapes you create on Spotify. You can
                    share them, subscribe to them, and collaborate on them with
                    your friends.
                  </h4>
                </div>
                <div class="grid-item">
                  <h2>Listen Offline</h2>
                  <h4>
                    Play music anywhere, even without internet! For Premium
                    subscription only.
                  </h4>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    )
  }
}

export default Features
