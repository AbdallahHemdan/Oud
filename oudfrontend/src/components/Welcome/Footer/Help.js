import React, { Component } from 'react';
import Navbar from '../Navbar/Navbar';
import LogoOud from './../../../assets/images/logo2.png';
import './../welcome.css';
class Help extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div className="container HelpSection">
          <img src={LogoOud} alt="" className="HelpLogo container" />
          <div className="Help_text">
            <h1>How can we help you?</h1>
            ACCOUNT HELP <a href="/forgot-password">Reset your password</a>
          </div>
        </div>

      </div>
    );
  }
}

export default Help;