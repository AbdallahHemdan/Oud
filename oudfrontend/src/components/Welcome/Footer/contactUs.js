import React, { Component } from 'react';
import Navbar from '../Navbar/Navbar';
import LogoOud from './../../../assets/images/logo2.png';
import Hemdan from './../../../assets/images/Hemdan.jpg';
import Ashraf from './../../../assets/images/Ashraf.jpg';
import Mahbob from './../../../assets/images/mahbob.jpg';
import Sedo from './../../../assets/images/sedo.png';
import Waled from './../../../assets/images/waled.jpg';
import './../welcome.css';
class ContactUs extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div className="container HelpSection">
          <img src={LogoOud} alt="" className="HelpLogo container" />
          <div className="Help_text">
            <h1>Contact with us</h1>
            <h2>Thanks goes to these wonderful people in the frontend team.</h2>
          </div>
          <div class="grid-container">
            <div class="item1">
              <div className="">
                <img src={Hemdan} className="ContactImage" alt="" />
                <h6>Abdallah Hemdan</h6>
                <a href="https://www.facebook.com/profile.php?id=100009270028400">
                  <i class="fab fa-facebook"></i>
                </a>
                <> </>
                <a href="https://github.com/AbdallahHemdan">
                  <i class="fab fa-github"></i>
                </a>
                <> </>
                <a href="https://www.linkedin.com/in/abdallah-a-hemdan-4a94a614a/">
                  <i class="fab fa-linkedin-in"></i>
                </a>
              </div>
            </div>
            <div class="item2">
              <div className="">
                <img src={Sedo} className="ContactImage" alt="" />
                <h6>Abdallah Abu Sedo</h6>
                <a href="https://www.facebook.com/abdallah.abusedo">
                  <i class="fab fa-facebook"></i>
                </a>
                <> </>
                <a href="https://github.com/abdallahabusedo">
                  <i class="fab fa-github"></i>
                </a>
                <> </>
                <a href="https://www.linkedin.com/in/abdallah-abu-sedo-63958a185/">
                  <i class="fab fa-linkedin-in"></i>
                </a>
              </div>
            </div>
            <div class="item3">
              <div className="">
                <img src={Waled} className="ContactImage" alt="" />
                <h6>Ahmed Walid</h6>
                <a href="https://www.facebook.com/profile.php?id=100007752348014">
                  <i class="fab fa-facebook"></i>
                </a>
                <> </>
                <a href="https://github.com/lido22">
                  <i class="fab fa-github"></i>
                </a>
                <> </>
                <a href="https://www.linkedin.com/in/abdallah-a-hemdan-4a94a614a/">
                  <i class="fab fa-linkedin-in"></i>
                </a>
              </div>
            </div>
            <div class="item4">
              <div className="">
                <img src={Mahbob} className="ContactImage" alt="" />
                <h6>Ahmed Mahboub </h6>
                <a href="https://www.facebook.com/ahmed.mahboub99">
                  <i class="fab fa-facebook"></i>
                </a>
                <> </>
                <a href="https://github.com/Mahboub99">
                  <i class="fab fa-github"></i>
                </a>
                <> </>
                <a href="https://www.linkedin.com/in/ahmed-mahboub-a6607a171/">
                  <i class="fab fa-linkedin-in"></i>
                </a>
              </div>
            </div>
            <div class="item5">
              <div className="">
                <img src={Ashraf} className="ContactImage" alt="" />
                <h6>Ahmed Ashraf</h6>
                <a href="https://www.facebook.com/aashrafh">
                  <i class="fab fa-facebook"></i>
                </a>
                <> </>
                <a href="https://github.com/aashrafh?fbclid=IwAR1wU3l6KmMtBYdMGQGEi1PvSFH68xOlgntLIs-_FjaStAgrAqq_rT_318k">
                  <i class="fab fa-github"></i>
                </a>
                <> </>
                <a href="https://www.linkedin.com/in/aashrafh/?fbclid=IwAR3cpLFGfeZ_CuPy21cE95pJZ9xc80CJgIEDyZSAhIYL0h-vi0PP9IUTzpk">
                  <i class="fab fa-linkedin-in"></i>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default ContactUs;