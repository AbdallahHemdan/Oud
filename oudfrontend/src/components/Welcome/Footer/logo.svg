<svg data-v-423bf9ae="" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 120.53311164128966 163" class="iconAbove"><!----><defs data-v-423bf9ae=""><linearGradient data-v-423bf9ae="" gradientTransform="rotate(25)" id="8e1cb018-c8d6-4530-9e34-9c81fbcaf2f3" x1="0%" y1="0%" x2="100%" y2="0%"><stop data-v-423bf9ae="" offset="0%" style="stop-color: rgb(9, 120, 166); stop-opacity: 1;"></stop><stop data-v-423bf9ae="" offset="100%" style="stop-color: rgb(251, 148, 71); stop-opacity: 1;"></stop></linearGradient></defs><!----><g data-v-423bf9ae="" id="784e0c9e-cb5e-48b4-8310-75f9c0fdcf6c" fill="#ffffff" transform="matrix(4.846526611095238,0,0,4.846526611095238,-5.3796433308556715,100.96445820601306)"><path d="M10.46 4.80L10.46 4.80L10.46 4.80Q10.46 5.59 10.14 6.59L10.14 6.59L10.14 6.59Q9.83 7.60 9.21 8.65L9.21 8.65L9.21 8.65Q8.60 9.70 7.79 10.59L7.79 10.59L7.79 10.59Q6.99 11.48 5.89 12.05L5.89 12.05L5.89 12.05Q4.79 12.61 3.62 12.61L3.62 12.61L3.62 12.61Q2.45 12.61 1.78 11.96L1.78 11.96L1.78 11.96Q1.11 11.31 1.11 10.16L1.11 10.16L1.11 10.16Q1.11 9.38 1.42 8.37L1.42 8.37L1.42 8.37Q1.74 7.36 2.35 6.31L2.35 6.31L2.35 6.31Q2.97 5.26 3.77 4.38L3.77 4.38L3.77 4.38Q4.58 3.50 5.68 2.93L5.68 2.93L5.68 2.93Q6.78 2.37 7.95 2.37L7.95 2.37L7.95 2.37Q9.11 2.37 9.79 3.01L9.79 3.01L9.79 3.01Q10.46 3.65 10.46 4.80ZM8.81 4.68L8.81 4.68L8.81 4.68Q8.81 3.14 7.69 3.14L7.69 3.14L7.69 3.14Q6.75 3.14 5.59 4.33L5.59 4.33L5.59 4.33Q4.44 5.52 3.63 7.20L3.63 7.20L3.63 7.20Q2.83 8.89 2.83 10.22L2.83 10.22L2.83 10.22Q2.83 11.76 3.95 11.76L3.95 11.76L3.95 11.76Q4.87 11.76 6.03 10.57L6.03 10.57L6.03 10.57Q7.20 9.38 8.00 7.69L8.00 7.69L8.00 7.69Q8.81 6.01 8.81 4.68ZM12.50 11.62L12.56 11.09L12.56 11.09Q12.56 10.99 12.49 10.99L12.49 10.99L12.46 10.99L12.25 11.09L12.25 11.09Q11.34 11.97 10.77 12.30L10.77 12.30L10.77 12.30Q10.19 12.63 9.58 12.63L9.58 12.63L9.58 12.63Q9.30 12.63 9.07 12.46L9.07 12.46L9.07 12.46Q8.85 12.29 8.85 11.82L8.85 11.82L8.85 11.82Q8.85 11.34 9.34 10.14L9.34 10.14L9.34 10.14Q9.83 8.93 10.56 7.83L10.56 7.83L10.56 7.83Q11.28 6.73 11.83 6.59L11.83 6.59L11.83 6.59Q12.01 6.54 12.32 6.54L12.32 6.54L12.32 6.54Q12.63 6.54 12.63 6.71L12.63 6.71L12.63 6.71Q12.63 6.76 12.12 7.48L12.12 7.48L12.12 7.48Q11.62 8.20 11.12 9.13L11.12 9.13L11.12 9.13Q10.63 10.05 10.63 10.51L10.63 10.51L10.63 10.51Q10.63 10.98 10.95 10.98L10.95 10.98L10.95 10.98Q11.55 10.98 12.33 10.19L12.33 10.19L12.33 10.19Q13.12 9.41 14.39 7.71L14.39 7.71L14.39 7.71Q14.74 7.21 15.51 7.21L15.51 7.21L15.51 7.21Q15.93 7.21 15.93 7.38L15.93 7.38L15.93 7.38Q15.93 7.42 15.89 7.49L15.89 7.49L15.89 7.49Q13.69 10.88 13.24 12.35L13.24 12.35L13.24 12.35Q13.19 12.52 13.10 12.52L13.10 12.52L13.06 12.52L13.06 12.52Q12.50 12.52 12.50 11.62L12.50 11.62ZM17.82 10.68L17.82 10.68L17.82 10.68Q18.56 10.02 19.12 9.29L19.12 9.29L19.12 9.29Q19.68 8.55 19.68 8.37L19.68 8.37L19.68 8.37Q19.68 8.18 19.52 8.05L19.52 8.05L19.52 8.05Q19.35 7.92 19.00 7.92L19.00 7.92L19.00 7.92Q18.19 7.92 17.29 9.03L17.29 9.03L17.29 9.03Q16.39 10.14 16.39 10.92L16.39 10.92L16.39 10.92Q16.39 11.34 16.69 11.34L16.69 11.34L16.69 11.34Q17.08 11.34 17.82 10.68ZM18.62 12.71L18.62 12.71L18.59 12.71L18.59 12.71Q17.92 12.71 17.92 11.96L17.92 11.96L17.95 11.68L17.85 11.54L17.85 11.54Q17.77 11.54 17.64 11.63L17.64 11.63L17.64 11.63Q16.44 12.80 15.53 12.80L15.53 12.80L15.53 12.80Q15.25 12.80 15.02 12.58L15.02 12.58L15.02 12.58Q14.78 12.36 14.78 11.83L14.78 11.83L14.78 11.83Q14.78 10.77 15.71 9.38L15.71 9.38L15.71 9.38Q17.16 7.18 19.38 7.18L19.38 7.18L19.38 7.18Q19.68 7.18 19.94 7.27L19.94 7.27L19.94 7.27Q20.20 7.36 20.27 7.36L20.27 7.36L20.27 7.36Q20.33 7.36 20.50 7.15L20.50 7.15L20.50 7.15Q20.66 6.93 21.57 5.38L21.57 5.38L21.57 5.38Q23.69 1.78 24.05 1.04L24.05 1.04L24.05 1.04Q24.36 0.42 25.61 0.42L25.61 0.42L25.61 0.42Q25.98 0.42 25.98 0.64L25.98 0.64L25.98 0.64Q25.98 0.73 25.39 1.51L25.39 1.51L25.39 1.51Q24.79 2.30 23.70 3.91L23.70 3.91L23.70 3.91Q22.60 5.52 21.66 7.21L21.66 7.21L21.66 7.21Q19.24 11.07 18.77 12.54L18.77 12.54L18.77 12.54Q18.72 12.71 18.62 12.71Z"></path></g><!----><g data-v-423bf9ae="" id="95b56a20-8d0c-4efd-8688-51bb49778216" transform="matrix(1.113988935166801,0,0,1.113988935166801,4.805272192390035,-10.095918140799412)" stroke="none" fill="#ffffff"><path d="M26.367 61.221a4.54 4.54 0 0 0 3.232-1.338 4.577 4.577 0 0 0 .001-6.465 4.54 4.54 0 0 0-3.231-1.338c-1.222 0-2.37.476-3.234 1.339a4.577 4.577 0 0 0 0 6.465 4.546 4.546 0 0 0 3.232 1.337zm-2.525-7.095a3.553 3.553 0 0 1 2.527-1.046c.954 0 1.85.371 2.524 1.045a3.577 3.577 0 0 1-.001 5.051c-1.349 1.348-3.701 1.349-5.05.001a3.577 3.577 0 0 1 0-5.051zM43.255 68.967c-.001 0 0 0 0 0-1.222 0-2.37.475-3.234 1.338a4.577 4.577 0 0 0 0 6.465 4.545 4.545 0 0 0 3.233 1.338 4.537 4.537 0 0 0 3.231-1.338 4.577 4.577 0 0 0 .001-6.465 4.532 4.532 0 0 0-3.231-1.338zm2.524 7.096c-1.348 1.349-3.702 1.347-5.051 0a3.577 3.577 0 0 1 0-5.051 3.552 3.552 0 0 1 2.527-1.045c.954 0 1.851.371 2.525 1.045a3.578 3.578 0 0 1-.001 5.051z"></path><path d="M87.743 19.44l-1.624-1.624c-.017-.017-.039-.024-.058-.038l1.508-1.667a.5.5 0 0 0-.017-.689l-3.069-3.071a.5.5 0 0 0-.689-.018l-1.658 1.5-1.475-1.476a1.83 1.83 0 0 0-.3-2.217c-.701-.701-1.924-.701-2.625 0a1.843 1.843 0 0 0-.545 1.313c0 .029.007.057.008.086a1.855 1.855 0 0 0-1.383.52 1.844 1.844 0 0 0-.544 1.313c0 .029.007.058.008.087-.504-.018-1.015.151-1.383.52a1.844 1.844 0 0 0-.544 1.313c0 .496.193.963.544 1.313s.817.544 1.313.544c.323 0 .63-.09.905-.244l1.248 1.247-1.629 1.475L53.72 39.184a.495.495 0 0 0-.24-.119c-.051-.007-1.283-.172-3.403-.172-6.156 0-21.964 1.49-35.768 15.295-6.543 6.544-6.377 17.358.37 24.105l6.931 6.931c3.341 3.341 7.743 5.181 12.396 5.181 4.452 0 8.611-1.707 11.711-4.809C63.515 67.8 60.869 46.635 60.84 46.423a.497.497 0 0 0-.119-.239l19.556-22.013 1.476-1.631 1.44 1.441a1.835 1.835 0 0 0-.245.906c0 .496.194.962.544 1.31.35.352.816.545 1.313.545a1.844 1.844 0 0 0 1.849-1.925c.023.001.046.007.069.007a1.852 1.852 0 0 0 1.85-1.926c.023.001.046.007.069.007.496 0 .962-.193 1.313-.544a1.86 1.86 0 0 0 0-2.627c-.577-.579-1.508-.671-2.212-.294zm-2.331-.917l1.62 1.62a1.855 1.855 0 0 0-.235.991 1.9 1.9 0 0 0-.973.225l-1.576-1.576 1.154-1.275c.004.005.005.011.01.015zm-1.3-5.128l2.397 2.398-6.62 7.316-1.469-1.469-.002-.003-.003-.002-.074-.073-1.547-1.548 7.318-6.619zM44.103 48.858a7.496 7.496 0 0 1 4.416 1.893L37.728 61.514c-2.652-2.997-2.553-7.591.315-10.458a7.536 7.536 0 0 1 6.06-2.198zm5.12 2.604c.027.031.058.06.084.092a7.506 7.506 0 0 1 1.74 4.253 7.555 7.555 0 0 1-2.197 6.055 7.591 7.591 0 0 1-5.403 2.237 7.582 7.582 0 0 1-5.009-1.879l10.785-10.758zm2.66 3.163c-.009-.041-.025-.079-.034-.119a8.52 8.52 0 0 0-.299-1.042c-.011-.03-.026-.06-.037-.091a8.657 8.657 0 0 0-.421-.932c-.031-.059-.06-.119-.092-.177a8.563 8.563 0 0 0-.575-.896c-.043-.06-.09-.117-.135-.176a9.268 9.268 0 0 0-.359-.438l6.995-6.977 2.432 2.433-7.475 8.415zm-2.656-4.58a8.41 8.41 0 0 0-.516-.431c-.059-.046-.116-.092-.176-.136a8.54 8.54 0 0 0-.895-.574c-.059-.033-.119-.062-.179-.093a8.286 8.286 0 0 0-.931-.421c-.03-.012-.059-.026-.089-.037a8.536 8.536 0 0 0-1.043-.299c-.041-.009-.078-.025-.119-.034l8.413-7.475 2.527 2.526-6.992 6.974zm5.213-10.164l21.608-19.198.409.409.899.899-20.43 20.375-2.486-2.485zm25.681-24.225l-1.379-1.379c.155-.275.245-.582.245-.906 0-.023-.006-.045-.007-.068.023.001.045.007.068.007.324 0 .631-.09.906-.245l1.439 1.44-1.272 1.151zm-1.931-4.203a.854.854 0 0 1 .857-.856.854.854 0 0 1 .606 1.461.877.877 0 0 1-1.212 0 .853.853 0 0 1-.251-.605zm-1.919 1.918a.854.854 0 0 1 .857-.855.854.854 0 0 1 .606 1.461.877.877 0 0 1-1.212 0 .853.853 0 0 1-.251-.606zm-1.667 2.525a.85.85 0 0 1-.251-.606c0-.229.089-.443.251-.605s.377-.251.606-.251.444.089.606.251.251.377.251.605a.85.85 0 0 1-.251.606.875.875 0 0 1-1.212 0zm2.218.3c.155-.274.245-.582.245-.906 0-.023-.006-.045-.007-.068.023.001.045.007.068.007.323 0 .631-.09.905-.244l1.344 1.344-1.272 1.151-1.283-1.284zM45.01 84.889c-2.912 2.912-6.82 4.516-11.004 4.516-4.386 0-8.537-1.735-11.689-4.888l-6.931-6.931c-6.357-6.357-6.523-16.537-.37-22.691 13.54-13.54 29.03-15.002 35.061-15.002 1.248 0 2.18.06 2.724.107l-8.84 7.854a8.554 8.554 0 0 0-6.625 2.495c-3.257 3.256-3.356 8.481-.316 11.871l-1.667 1.662a8.607 8.607 0 0 0-4.981-2.204c-.133-.013-.272.055-.375.145a.455.455 0 0 0-.09.109 6.242 6.242 0 0 1-4.379 1.776c-.335 0-.671-.026-1.047-.087a.643.643 0 0 0-.06-.013l-.108-.018v.002l-.011-.002a.502.502 0 0 0-.551.357 4.574 4.574 0 0 0 3.622 5.767c-.035.586.14 1.164.48 1.647l-1.67 1.666-4.451-4.451c-.702-.701-1.925-.701-2.627 0a1.859 1.859 0 0 0 0 2.626l9.595 9.595c.351.351.817.544 1.314.544s.963-.193 1.313-.544a1.863 1.863 0 0 0-.001-2.627l-4.436-4.436 1.681-1.676c.46.294 1.004.438 1.554.406a4.57 4.57 0 0 0 1.274 2.461 4.54 4.54 0 0 0 3.232 1.339 4.6 4.6 0 0 0 1.262-.177.5.5 0 0 0 .355-.563l-.02-.116a6.299 6.299 0 0 1 1.631-5.431.5.5 0 0 0 .288-.656 8.63 8.63 0 0 0-2.091-4.731l1.672-1.668a8.582 8.582 0 0 0 5.717 2.173 8.58 8.58 0 0 0 6.11-2.53 8.543 8.543 0 0 0 2.495-6.626l7.851-8.837c.276 3.371.976 21.922-14.891 37.787zm-9.822-9.665a3.718 3.718 0 0 1-.559.043 3.544 3.544 0 0 1-2.525-1.046c-.629-.63-1-1.466-1.042-2.354a.511.511 0 0 0-.465-.475l-.045-.002a.5.5 0 0 0-.138.02c-.366.105-.755.06-1.098-.094l6.016-6c.972 1.131 1.615 2.507 1.804 3.975a7.294 7.294 0 0 0-1.948 5.933zm-6.759-5.798a.581.581 0 0 0 .02-.149.5.5 0 0 0-.476-.499 3.574 3.574 0 0 1-3.356-4.126 7.281 7.281 0 0 0 5.931-1.948c1.518.193 2.937.87 4.086 1.896l-6.062 6.046a1.598 1.598 0 0 1-.143-1.22zm2.189 9.454a.858.858 0 0 1 0 1.212.878.878 0 0 1-1.212 0l-9.595-9.595a.858.858 0 0 1 1.212-1.212l9.595 9.595zm29.405-33.417l-2.39-2.39 20.43-20.375 1.156 1.157-19.196 21.608zm23.553-24.938l1.537 1.537a1.847 1.847 0 0 0-.235.992 1.894 1.894 0 0 0-.972.225l-1.481-1.481 1.151-1.273zm2.086 4.361a.85.85 0 0 1-.25.605.875.875 0 0 1-1.212-.001.847.847 0 0 1-.251-.604c0-.229.089-.443.252-.605.162-.162.377-.251.606-.251s.444.089.605.25.25.377.25.606zm1.668-1.313a.88.88 0 0 1-1.211 0 .86.86 0 0 1 .605-1.463.857.857 0 0 1 .606 1.463zm1.919-1.919a.878.878 0 0 1-1.211.001.861.861 0 0 1 0-1.213.85.85 0 0 1 .606-.251.859.859 0 0 1 .605 1.463z"></path></g><!----></svg>