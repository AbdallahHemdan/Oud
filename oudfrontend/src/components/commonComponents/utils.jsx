import axios from "axios";
import { base } from "../../config/environment"
import {config} from "../../utils/auth"
/**
 * @file this file contains three functions used in Playlist, Album, LikedSongs 
 **/
export function addSong(playlistId, track){
  axios
  .post(`${base}/playlists/${playlistId}/${track}`, config)
  .then((response) => {
  })
  .catch((error) => {
  });
}