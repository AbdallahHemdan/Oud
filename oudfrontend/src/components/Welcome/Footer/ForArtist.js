import React, { Component } from 'react';
import Navbar from '../Navbar/Navbar';
import './../welcome.css';
import Footer from './Footer';
import Fimage from '../../../assets/images/Kota_Profile.webp';
import Simage from './../../../assets/images/Kota_Artist_Pick.webp';
import Timage from './../../../assets/images/Kota_Stat.webp';
class ForArtist extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div className="container HelpSection" id="gohere">
          <div className="Help_text">
            <p className="FAHead">Make OUD Yours.</p>
            <hr className="hrinFA" />
          </div>
          <div>
            <p className="Text1">
              25+ fans are waiting for you. Claim your profile to access OUD for
              Artists.
              <a href="/signup">
                <button className="GETBTN">GET ACCESS</button>
              </a>
            </p>
          </div>
          <div class="grid-container2">
            <div class="item1">
              <img src={Fimage} alt="" className="FAImage"></img>
              <h2>Tell Your Story.</h2>
              <h6>
                Introduce yourself to listeners with your full profile and
                Artist's Pick.
              </h6>
            </div>
            <div class="item2">
              <img src={Simage} alt="" className="FAImage"></img>
              <h2>Promote your music.</h2>
              <h6>
                Pitch to playlists, release new music, and find new followers.
              </h6>
            </div>
            <div class="item3">
              <img src={Timage} alt="" className="FAImage"></img>
              <h2>Measure your noise.</h2>
              <h6>
                Dig into song stats and fan insights to see what's hitting
                hardest.
              </h6>
            </div>
          </div>
        </div>
        <div className="TSEction">
          <p className="FAHead">Let’s go.</p>
          <p className="TSEText">
            If you’re an artist or part of their management team, we’ll show you
            how to get the most out of OUD.
          </p>
          <div className="container">
            <a href="/signup">
              <button className="GETBTN2" to="/signin">
                GET ACCESS
              </button>
            </a>
          </div>
        </div>
        <Footer />
      </div>
    );
  }
}

export default ForArtist;