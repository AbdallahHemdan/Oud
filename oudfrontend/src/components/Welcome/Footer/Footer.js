import React from 'react';
import { Link } from 'react-router-dom';
import logo from './../../../assets/images/Logoc.png';
import './../welcome.css';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faFacebook } from '@fortawesome/free-brands-svg-icons';
import { faTwitter } from '@fortawesome/free-brands-svg-icons';
import { faInstagram } from '@fortawesome/free-brands-svg-icons';
import { faYoutube } from '@fortawesome/free-brands-svg-icons';
/**
 * the footer function
 * @function
 * @returns {JSX}
 */
function Footer() {
  return (
    <div>
      <footer className="main-footer">
        <div className="footer-middle Ffooter-middle">
          <div className="container">
            <div className="row">
              {/* Column 1 */}
              <div className="col-md-3 col-sm-6">
                <Link to="/welcomeUser">
                  <img
                    data-testid="logoImage"
                    className="img-responsive img-resp "
                    src={logo}
                    alt="Oud logo"
                  />
                </Link>
              </div>
              {/* Column 2 */}
              <div className="col-md-3 col-sm-6">
                <h5
                  data-testid="Company"
                  className="font-weight-bold text-uppercase mt-3 mb-4"
                >
                  Company
                </h5>
                <ul className="list-unstyled ListMain hovergold ">
                  <li className="ListItem">
                    <Link data-testid="About" to="/about">
                      About
                    </Link>
                  </li>
                  <li className="ListItem">
                    <Link data-testid="Features" to="/features">
                      Features
                    </Link>
                  </li>
                  <li className="ListItem">
                    <Link data-testid="Help" to="/help">
                      Help
                    </Link>
                  </li>
                </ul>
              </div>
              {/* Column 3 */}
              <div className="col-md-3 col-sm-6">
                <h5
                  data-testid="Others"
                  className="font-weight-bold text-uppercase mt-3 mb-4"
                >
                  Others
                </h5>

                <ul className="list-unstyled ListMain hovergold">
                  <li className="ListItem">
                    <Link data-testid="Artists" to="/forartists">
                      For Artists
                    </Link>
                  </li>
                  <li className="ListItem">
                    <Link data-testid="contact_us" to="/contactus">
                      contact us
                    </Link>
                  </li>
                </ul>
              </div>
              {/* Column 4 */}
              <div className="col-md-3 col-sm-6">
                <h5
                  data-testid="Links"
                  className="font-weight-bold text-uppercase mt-3 mb-4"
                >
                  Links
                </h5>

                <ul className="list-unstyled ListMain hovergold">
                  <li className="ListItem">
                    <Link data-testid="WePlayer" to="/">
                      Web Player
                    </Link>
                  </li>
                  <li className="ListItem">
                    <Link data-testid="MobileApp" to="/android">
                      Free Mobile App
                    </Link>
                  </li>
                </ul>
              </div>
              {/* Column 5 */}
              <div className="col-md-3 col-sm-6 ml-auto">
                <ul className="list-unstyled ListMain list-inline text-center">
                  <li
                    data-testid="faFacebook"
                    className="list-inline-item fa-2x "
                  >
                    <FontAwesomeIcon icon={faFacebook} />
                  </li>
                  <li
                    data-testid="faTwitter"
                    className="list-inline-item fa-2x"
                  >
                    <FontAwesomeIcon icon={faTwitter} />
                  </li>
                  <li
                    data-testid="faInstagram"
                    className="list-inline-item fa-2x"
                  >
                    <FontAwesomeIcon icon={faInstagram} />
                  </li>
                  <li
                    data-testid="faYoutube"
                    className="list-inline-item fa-2x"
                  >
                    <FontAwesomeIcon icon={faYoutube} />
                  </li>
                </ul>
              </div>
            </div>
            {/* Footer Bottom */}
            <div className="footer-bottom Ffooter-bottom">
              <p className="text-xs-center">
                &copy;{new Date().getFullYear()} Oud - All Rights Reserved
              </p>
            </div>
          </div>
        </div>
      </footer>

      <div></div>
    </div>
  );
}
export default Footer;
