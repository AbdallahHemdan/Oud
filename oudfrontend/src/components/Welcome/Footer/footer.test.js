import React from 'react';
import {shallow} from 'enzyme';
import {findByTestAttr} from './../../../utils/index';
import Footer from './Footer';
const setup = (props = {}) => {
  const component = shallow(<Footer {...props} />);
  return component;
};

describe('Body page testing ', () => {
  let component;
  beforeEach(() => {
    const props = {};
    component = setup(props);
  });
  it('render correctly text component', () => {
    const TextInputComponent = shallow(<Footer />);
    expect(TextInputComponent).toMatchSnapshot();
  });

  it('render a logoImage ', () => {
    const wrapper = findByTestAttr(component, 'logoImage');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Company', () => {
    const wrapper = findByTestAttr(component, 'Company');
    expect(wrapper.length).toBe(1);
  });

  it('render a Text About', () => {
    const wrapper = findByTestAttr(component, 'About');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Features', () => {
    const wrapper = findByTestAttr(component, 'Features');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Help', () => {
    const wrapper = findByTestAttr(component, 'Help');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Others', () => {
    const wrapper = findByTestAttr(component, 'Others');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Artists', () => {
    const wrapper = findByTestAttr(component, 'Artists');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text Links', () => {
    const wrapper = findByTestAttr(component, 'Links');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text WePlayer', () => {
    const wrapper = findByTestAttr(component, 'WePlayer');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text MobileApp', () => {
    const wrapper = findByTestAttr(component, 'MobileApp');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text faFacebook', () => {
    const wrapper = findByTestAttr(component, 'faFacebook');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text faTwitter', () => {
    const wrapper = findByTestAttr(component, 'faTwitter');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text faInstagram', () => {
    const wrapper = findByTestAttr(component, 'faInstagram');
    expect(wrapper.length).toBe(1);
  });
  it('render a Text faYoutube', () => {
    const wrapper = findByTestAttr(component, 'faYoutube');
    expect(wrapper.length).toBe(1);
  });
});
