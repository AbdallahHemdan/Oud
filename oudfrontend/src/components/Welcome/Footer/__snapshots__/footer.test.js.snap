// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body page testing  render correctly text component 1`] = `ShallowWrapper {}`;
